src/main/kotlin/com/sourcegraph/cody/agent/CodyAgentService.kt

```diff
@@ -77,11 +77,15 @@ class CodyAgentService(project: Project) : Disposable {
       }
 
       agent.client.onOpenUntitledDocument = Function { params ->
-        val uri = URI.create(params.uri).withScheme("file")
-        if (CodyEditorUtil.createFileIfNeeded(project, uri, params.content) == null)
+        val uri = URI.create(params.uri)
+        if (!uri.path.contains("/")) {
```

Comment:
Is it worth commenting why? IIUC it is because we produce VirtualFiles, and below `uriFor` ensures they start with `/`?

With that being the case, can you harden this assertion that `uri.path[0]` is `'/'`? Should we log here? Could we throw here instead?

Reply:
Or even better `startsWith("/")` to avoid crashes in case of empty uri string.